app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodFacade
+  def self.search(keyword)
+    result = FoodService.search(keyword)
+    result[:foods][0..9].map do |data|
```

Comment:
Is there another way you can limit the number of foods retrieved without using ruby logic?